mteb/models/model_implementations/slm_models.py

```diff
@@ -0,0 +1,454 @@
+    modalities=["image", "text"],
+    n_parameters=1_700_000_000,
+    memory_usage_mb=3400,
+    max_tokens=262144,
+    embed_dim=128,
+    license="apache-2.0",
+    open_weights=True,
+    public_training_code=None,
+    public_training_data=None,
+    framework=["ColPali"],
+    reference="https://huggingface.co/VAGOsolutions/SauerkrautLM-ColQwen3-1.7b-Turbo-v0.1",
+    similarity_fn_name=ScoringFunction.MAX_SIM,
+    use_instructions=True,
+    adapted_from="Qwen/Qwen3-VL-2B-Instruct",
+    training_datasets={"vidore/colpali_train_set"},
+    citation=SAUERKRAUTLM_CITATION + COLPALI_CITATION,
+)
+
+# ColQwen3-2B: ~2.2B params → 4.4 GB VRAM in bfloat16
+slm_colqwen3_2b = ModelMeta(
+    loader=SLMColQwen3Wrapper,
+    name="VAGOsolutions/SauerkrautLM-ColQwen3-2b-v0.1",
+    languages=SUPPORTED_LANGUAGES,
+    revision="48f699713c10af754684e12060a2af9266462cc9",
+    release_date="2025-12-20",
+    modalities=["image", "text"],
+    n_parameters=2_200_000_000,
+    memory_usage_mb=4400,
+    max_tokens=262144,
+    embed_dim=128,
+    license="apache-2.0",
+    open_weights=True,
+    public_training_code=None,
+    public_training_data=None,
+    framework=["ColPali"],
+    reference="https://huggingface.co/VAGOsolutions/SauerkrautLM-ColQwen3-2b-v0.1",
+    similarity_fn_name=ScoringFunction.MAX_SIM,
+    use_instructions=True,
+    adapted_from="Qwen/Qwen3-VL-2B-Instruct",
+    training_datasets={"vidore/colpali_train_set"},
+    citation=SAUERKRAUTLM_CITATION + COLPALI_CITATION,
+)
+
+# ColQwen3-4B: ~4B params → 8 GB VRAM in bfloat16
+slm_colqwen3_4b = ModelMeta(
+    loader=SLMColQwen3Wrapper,
+    name="VAGOsolutions/SauerkrautLM-ColQwen3-4b-v0.1",
+    languages=SUPPORTED_LANGUAGES,
+    revision="b635fbb3ab145f07608ed10a85def33544de1723",
+    release_date="2025-12-20",
+    modalities=["image", "text"],
+    n_parameters=4_000_000_000,
+    memory_usage_mb=8000,
+    max_tokens=262144,
+    embed_dim=128,
+    license="apache-2.0",
+    open_weights=True,
+    public_training_code=None,
+    public_training_data=None,
+    framework=["ColPali"],
+    reference="https://huggingface.co/VAGOsolutions/SauerkrautLM-ColQwen3-4b-v0.1",
+    similarity_fn_name=ScoringFunction.MAX_SIM,
+    use_instructions=True,
+    adapted_from="Qwen/Qwen3-VL-4B-Instruct",
+    training_datasets={"vidore/colpali_train_set"},
+    citation=SAUERKRAUTLM_CITATION + COLPALI_CITATION,
+)
+
+# ColQwen3-8B: ~8B params → 16 GB VRAM in bfloat16
+slm_colqwen3_8b = ModelMeta(
+    loader=SLMColQwen3Wrapper,
+    name="VAGOsolutions/SauerkrautLM-ColQwen3-8b-v0.1",
+    languages=SUPPORTED_LANGUAGES,
+    revision="36ac136e451a7b8d8229725d69d4ec23aa4f03c8",
+    release_date="2025-12-20",
+    modalities=["image", "text"],
+    n_parameters=8_000_000_000,
+    memory_usage_mb=16000,
+    max_tokens=262144,
+    embed_dim=128,
+    license="apache-2.0",
+    open_weights=True,
+    public_training_code=None,
+    public_training_data=None,
+    framework=["ColPali"],
```

Comment:
You should add to all models
```suggestion
    framework=["ColPali"],
    model_type=["late-interactiion"],
```